client/src/Utils/api.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Contains functions which call the back-end
+ */
+
+import axios from "axios";
+
+/*
+ * Check if current user has given permission for Mail Sender to access their
+ * gmail account
+ */
+const checkForGmailToken = async () => {
+  const response = await axios.get(`/api/gmail-auth/checkToken`).catch(err => {
```

Comment:
if you are using await, you should use a try catch block instead of mixing the promise into it too

Reply:
Good point. Fixed. Thanks